assets/css/edd-admin-chosen.css

```diff
@@ -203,10 +193,13 @@
 /* WordPress Color Schemes */
 
 body.admin-color-fresh .chosen-container-active .chosen-single,
-body.admin-color-fresh .chosen-container-active .chosen-choices {
+body.admin-color-fresh .chosen-container-active .chosen-choices,
+body.admin-color-fresh .chosen-with-drop.chosen-dropup .chosen-drop {
 	border-color: #007cba;
 	box-shadow: 0 0 0 1px #007cba;
 }
+body.admin-color-fresh ..chosen-with-drop.chosen-dropup .chosen-single,
```

Comment:
Double period is causing styling issues with Fresh color scheme:

<img width="336" alt="jjj-sc-2020-07-27 at 12 21 24@2x" src="https://user-images.githubusercontent.com/88951/88571729-b5b15b00-d003-11ea-984b-998d875a0665.png">